webapp/src/component/security/OAuthService.tsx

```diff
@@ -58,13 +58,18 @@ export const oauth2Service = (
   const redirectUri = LINKS.OAUTH_RESPONSE.buildWithOrigin({
     [PARAMS.SERVICE_TYPE]: 'oauth2',
   });
+  const authUrl = new URL(authorizationUrl);
+  authUrl.searchParams.set('client_id', clientId);
+  authUrl.searchParams.set('redirect_uri', redirectUri);
+  authUrl.searchParams.set('response_type', 'code');
+  authUrl.searchParams.set(
+    'scope',
+    scopes.map((scope) => `${scope}`).join('·')
```

Comment:
This `.map` looks a bit weird, I don't think it's necessary. But the real problem is with the join, there seem to be an unexpected character `·` instead of a plain space.

Reply:
Oops...missed that one when copying the format from the prettier output in the earlier build (because I was too lazy to install dependencies and run prettier locally 😬).

As far as having the map at all, I only left it because we had it in there from before, but I'm happy to take it out if you want and just make it a `scopes.join(' ')`.

Reply:
Updates have been made, and I ran prettier locally this time ... 3rd times the charm 😄